bin/udb3.php

```diff
@@ -52,106 +20,66 @@
-$consoleApp->add(new FireProjectedToJSONLDCommand($app[EventBus::class], $app[OrganizerJSONLDServiceProvider::JSONLD_PROJECTED_EVENT_FACTORY], $app[PlaceJSONLDServiceProvider::JSONLD_PROJECTED_EVENT_FACTORY]));
-$consoleApp->add(
-    new ProcessDuplicatePlaces(
-        $app['event_command_bus'],
-        $app['duplicate_place_repository'],
-        $app['canonical_service'],
-        $app[EventBus::class],
-        $app[PlaceJSONLDServiceProvider::JSONLD_PROJECTED_EVENT_FACTORY],
-        $app[EventRelationsRepository::class],
-        $app['dbal_connection']
-    )
-);
-$consoleApp->add(
-    new ReindexOffersWithPopularityScore(
-        OfferType::event(),
-        $app['dbal_connection'],
-        $app[EventBus::class],
-        $app[EventJSONLDServiceProvider::JSONLD_PROJECTED_EVENT_FACTORY]
-    )
-);
-$consoleApp->add(
-    new ReindexOffersWithPopularityScore(
-        OfferType::place(),
-        $app['dbal_connection'],
-        $app[EventBus::class],
-        $app[PlaceJSONLDServiceProvider::JSONLD_PROJECTED_EVENT_FACTORY]
-    )
-);
-$consoleApp->add(
-    new ReindexEventsWithRecommendations(
-        $app['dbal_connection'],
-        $app[EventBus::class],
-        $app[EventJSONLDServiceProvider::JSONLD_PROJECTED_EVENT_FACTORY]
-    )
-);
-$consoleApp->add(new UpdateOfferStatusCommand(OfferType::event(), $app['event_command_bus'], $app[Sapi3SearchServiceProvider::SEARCH_SERVICE_EVENTS]));
-$consoleApp->add(new UpdateOfferStatusCommand(OfferType::place(), $app['event_command_bus'], $app[Sapi3SearchServiceProvider::SEARCH_SERVICE_PLACES]));
-$consoleApp->add(new UpdateBookingAvailabilityCommand($app['event_command_bus'], $app[Sapi3SearchServiceProvider::SEARCH_SERVICE_EVENTS]));
-$consoleApp->add(new UpdateEventsAttendanceMode($app['event_command_bus'], $app[Sapi3SearchServiceProvider::SEARCH_SERVICE_EVENTS]));
-$consoleApp->add(new ChangeOfferOwner($app['event_command_bus']));
-$consoleApp->add(new ChangeOfferOwnerInBulk($app['event_command_bus'], $app['offer_owner_query']));
-$consoleApp->add(new ChangeOrganizerOwner($app['event_command_bus']));
-$consoleApp->add(new ChangeOrganizerOwnerInBulk($app['event_command_bus'], $app['organizer_owner.repository']));
-$consoleApp->add(new UpdateUniqueLabels($app['dbal_connection']));
-$consoleApp->add(new UpdateUniqueOrganizers($app['dbal_connection'], new WebsiteNormalizer()));
-$consoleApp->add(new RemoveFacilitiesFromPlace($app['event_command_bus'], $app[Sapi3SearchServiceProvider::SEARCH_SERVICE_PLACES]));
-$consoleApp->add(new RemoveLabelOffer($app['dbal_connection'], $app['event_command_bus']));
-$consoleApp->add(new RemoveLabelOrganizer($app['dbal_connection'], $app['event_command_bus']));
-
-$consoleApp->add(new ImportOfferAutoClassificationLabels($app['dbal_connection'], $app['event_command_bus']));
-
-$consoleApp->add(new ReplaceNewsArticlePublisher($app['dbal_connection']));
+$commands = [
```

Comment:
Would it be an option to expose the service list from `ConsoleServiceProvider` and use that to generate the command map?

Reply:
Yes, good idea!

Reply:
See [75de095](https://github.com/cultuurnet/udb3-silex/pull/1171/commits/75de095f7b2f49b1f57c305be6845f187d3292a4)

Reply:
Nice approach!